sdk/digitaltwins/ci.yml

```diff
@@ -29,4 +29,6 @@ extends:
     ArtifactName: packages
     Artifacts:
     - name: Azure.DigitalTwins.Core
-      safeName: AzureDigitalTwinsCore
\ No newline at end of file
+      safeName: AzureDigitalTwinsCore
```

Comment:
@weshaggard 

Am I doing this right? The build doesn't seem to be producing artifacts now.

```
Starting: Download Build Artifacts
==============================================================================
Task         : Download Pipeline Artifacts
Description  : Download build and pipeline artifacts
Version      : 2.3.1
Author       : Microsoft Corporation
Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/download-pipeline-artifact
==============================================================================
Download from the specified build: #432014
Download artifact to: d:\a\1/packages
##[error]Artifact packages was not found for build 432014.
Finishing: Download Build Artifacts
```

Reply:
This part looks correct. There are no packages because the build/package step failed.